apcd-cms/src/apps/components/paginator/static/paginator/css/paginator.css

```diff
@@ -0,0 +1,70 @@
+/*! @tacc/core-styles 1.0.0+ | MIT | github.com/TACC/Core-Styles */
```

Comment:
**Superfluous Style Load?** The file `apcd-cms/src/apps/components/paginator/templates/paginator.html` uses `c-page` classes. **If** the Bootstrap `pagaination` class are not used at all, **then** please only load the Core-Styles for the markup you use:

https://unpkg.com/@tacc/core-styles@0.10.0/dist/components/c-page.css